css-two-column-layout/styles.css

```diff
@@ -0,0 +1,61 @@
+* {
+  font-family: Karla, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+}
+
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+  margin: 25px auto;
+  justify-content: center;
+}
+
+.column {
+  text-align: center;
+}
+
+.column-half {
+  width: calc(85% / 2);
```

Comment:
remember that column classes should have widths which semantically match their class name

Reply:
![Screenshot 2022-05-26 145138two-column-layout](https://user-images.githubusercontent.com/90532731/170594155-15a6cb93-ff35-4465-bcee-da29f9f74751.png)